src/AbstractFixerSuite.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
I do not see any value of this class, I could use `Config::addCustomFixers` without it.

When I want to provide a nice class for my set of custom fixers I will still not need it. I will just create a class based on Traversable (eg IteratorAggregate). Also, if at all, then I would go in Interface direction. But then even less value of it.
Maybe the class-level docs would be a nice place to show value of this class.